lib/components/molecules/CookieDisclaimer/CookieDisclaimer.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import Button from '../../atoms/Button';
+import Icon from '../../atoms/Icon';
+import type { cookieDisclaimerProps } from './types';
+import styles from './CookieDisclaimer.style';
+
+let [isVisible, setIsVisible] = [true, null];
```

Comment:
Shouldn't this be with-in the component? Why keep it global to module?

Reply:
I have to pass these on every actionable items to update the state as Close, Reject and Accept.